digits/model/tasks/caffe_train.py

```diff
@@ -807,6 +813,26 @@ def task_arguments(self, resources, env):
         if self.pretrained_model:
             args.append('--weights=%s' % ','.join(map(lambda x: self.path(x), self.pretrained_model.split(':'))))
 
+        if platform.system() == 'Windows' and including_python_layer:
+            if len(identifiers) > 1:
+                    self.logger.warning('Python layer can not work in multiple GPU.  Will use the first GPU only')
```

Comment:
Yes.  The framework is the best place to handle issue 1.  I will modify the codes not to check multi-GPU case.